curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-027.md

```diff
@@ -0,0 +1,177 @@
+---
+id: 6143cb26f7edff2dc28f7da5
+title: Part 27
+challengeType: 0
+dashedName: part-27
+---
+
+# --description--
+
+Each `fieldset` will contain a true/false question.
+
+Within each `fieldset`, nest one `legend` element, and one `ul` element with two options.
+
+# --hints--
+
+You should nest one `legend` element within the first `fieldset` element.
+
+```js
+assert.equal(document.querySelectorAll('fieldset:nth-of-type(1) > legend')?.length, 1);
```

Comment:
```html
        <div class="question-block">
          <p>1</p>
          <fieldset class="question">
            <legend class="one"></legend>
            <ul>
              <li></li>
              <li></li>
            </ul>
          </fieldset>
        </div>
        <div class="question-block">
          <p>2</p>
          <fieldset class="question">
            <legend class="two"></legend>
            <ul>
              <li></li>
              <li></li>
            </ul>
          </fieldset>
        </div>
```

With this code, the test gives me back both `legends`:

![image](https://user-images.githubusercontent.com/63889819/134577600-7eb89581-6bc7-4fa3-a124-e87fe2164185.png)